docs/GettingStartedDocs/best_practices_secure_enclaves.md

```diff
@@ -0,0 +1,275 @@
+# Best Practices for Keeping Enclaves Secure
+Use this guide to apply secure patterns and avoid common mistakes that put the security of enclave applications at risk.
+
+1.  [Best Practices for Interface Custom Marshaling ](#1.-best-practices-for-interface-custom-marshaling)
+
+    1.1 [Ensuring memory is where it should be](#1.1-ensuring-memory-is-where-it-should-be)
+
+    1.2 [SGX-based enclaves: oe_is_within_enclave != !oe_is_outside_enclave](#1.2-SGX-based-enclaves:-oe_is_within_enclave-!=-!oe_is_outside_enclave)
```

Comment:
There's an incompatibility between Visual Code and Github's markdown:  Github's parser does not like something in the link. I will have to experiment more tomorrow.

Reply:
Finally!